src/Footer/__snapshots__/Footer.test.jsx.snap

```diff
@@ -1,31 +1,33 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`Legal Matches snapshot 1`] = `
-<footer
-  className="footer"
-  data-tracking="Global footer"
->
-  <div
-    className="container"
+<Fragment>
```

Comment:
Not sure this test will pass will it? We're not returning a fragment here

Reply:
updated the snapshot, seems to have sorted it.